sktime/performance_metrics/forecasting/_classes.py

```diff
@@ -642,7 +642,7 @@ def custom_mae(y_true, y_pred) -> float:
 
             return float(result)
 
-        params2 = {"func": custom_mae, "name": "custom_nmae", "lower_is_better": True}
+        params2 = {"func": custom_mae, "name": "custom_mae", "lower_is_better": True}
```

Comment:
@fkiraly This is a hard error.

Reply:
This is undesirable for sure, and should be fixed, but why is this a "hard error"?

It's a text describing what's the metric supposed to measure, but what is the effect of having a wrong text in metric description that is only being used aas part of unit tests? Please let me know if I am missing something.

Reply:
I thought "name" is later processed.
This was a mistake.